src/api/db/queries/create-table-object-hashes.ts

```diff
@@ -0,0 +1,7 @@
+export const createTableObjectHashesQuery = `
+  CREATE TABLE object_hashes (
```

Comment:
我之前的意思是将 hashes 从 table name 中去掉，去掉后 file 可能和 File 有点重复，所以我想的就是对齐 object。这里处理成什么样，你决定吧（保留目前的你可以自己 resolve 一下哇）
```suggestion
  CREATE TABLE objects (
```